quickwit/quickwit-serve/src/rest.rs

```diff
@@ -37,15 +39,19 @@ use crate::search_api::{search_get_handler, search_post_handler, search_stream_h
 use crate::ui_handler::ui_handler;
 use crate::{Format, QuickwitServices};
 
-/// Starts REST service given a HTTP address and a search service.
+/// Starts REST services.
 pub(crate) async fn start_rest_server(
     rest_listen_addr: SocketAddr,
+    universe: &Universe,
     quickwit_services: &QuickwitServices,
 ) -> anyhow::Result<()> {
     info!(rest_listen_addr = %rest_listen_addr, "Starting REST server.");
     let request_counter = warp::log::custom(|_| {
         crate::SERVE_METRICS.http_requests_total.inc();
     });
+
+    let delete_task_service_opt: Option<Mailbox<DeleteTaskService>> = universe.get_one();
```

Comment:
it's a bit weird to have this here alone, but I see the rationale: `delete_task_service_opt` is not a service as important as the others. So you did not include it in the health checks right? Thus this `get_one`.

Reply:
and now we also return a janitor service so it's harder to get this delete service. okokok